libs/components/code-examples/src/lib/modules/forms/checkbox/icon-group/example.component.html

```diff
@@ -1,30 +1,28 @@
-<div class="sky-padding-even-xl">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.